### DISCUSSION TITLE
Lovelace card with touch-friendly controls

### DISCUSSION THREAD

**Discussion by lamixer** [nathanmarlor/foxess_modbus#717] [Show and tell]
(edit - my solution below!)

Hello!

I'm working on a little RPi-based kisok control panel. I'd like to be able to tap to input (increment or decrement) values for things like number.foxess_max_charge_current. The normal suggested card that I'm using has up/down arrows to the right of the box showing the current numeric setting, but it's too small to tap it with a finger with my set up. Any suggested add-on cards for this? Thanks!

![standard-lovelace](https://github.com/user-attachments/assets/8bd40a81-db97-4eb3-8b43-b4c95beac503)

**Comment by lamixer** [comment]
Ok, I found something. I used HACS to install "Custom Features for Home Assistant Cards". Then I made a vertical stack, and put each entry I wanted to control in the stack using the new features that this plug-in allowed. Each card has a config something like this:

```  - type: tile
    entity: number.foxess_force_charge_power
    show_entity_picture: false
    tap_action:
      action: none
    icon_tap_action:
      action: none
    features:
      - type: custom:service-call
        entries:
          - type: slider
            entity_id: number.foxess_force_charge_power
            range:
              - 0.5
              - 8
            tap_action:
              action: perform-action
              perform_action: number.set_value
              data:
                value: "{{ value | float }}"
              target:
                entity_id: number.foxess_force_charge_power
              confirmation: true
            step: 0.5
            value_from_hass_delay: 1500
```

I found better behaviour by using the confirmation: true config option.

Result looks like this!

![with-new-card-enhancements](https://github.com/user-attachments/assets/7bf0fb45-31f1-493b-94b5-75872d7f138e)

**Reply by FozzieUK** [reply]
Very nice, good spot👍
